panorama_connector.py

```diff
@@ -905,7 +918,9 @@ def _unblock_application(self, param):
 
         message = action_result.get_message()
 
-        self._commit_and_commit_all(param, action_result)
+        status = self._commit_and_commit_all(param, action_result)
+        if phantom.is_fail(status):
+            return action_result.get_status()
```

Comment:
We now return the error status as soon as it takes place. If it gets past here, our current Success messages would be accurate.